### PR TITLE
Update precommit test file list to run all test files.

### PR DESCRIPTION
**Pull request by LaszloLango**
Issue: #716

JerryScript-DCO-1.0-Signed-off-by: László Langó llango.u-szeged@partner.samsung.com

### CODE REVIEW

File: tests/jerry-test-suite/11/11.04/11.04.06/11.04.06-007.js [hunk 1]
```diff
@@ -13,4 +13,4 @@
 // limitations under the License.
 
 var a = new Number(0);
-assert(+a === 0) ? 1 : 0;
\ No newline at end of file
+assert(+a === 0) ? true : false;
```
Comment (ruben-ayrapetyan) [inline tests/jerry-test-suite/11/11.04/11.04.06/11.04.06-007.js]: `? true : false` is not necessary outside of `assert`'s condition

File: tests/jerry-test-suite/15/15.05/15.05.02/15.05.02.01/15.05.02.01-001.js [hunk 2]
```diff
@@ -12,4 +12,4 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-assert(String.prototype.isPrototypeOf(new String("")) ? 1 : 0);
+assert(String.prototype.isPrototypeOf(new String("")) ? true : false);
```
Comment (ruben-ayrapetyan) [inline tests/jerry-test-suite/15/15.05/15.05.02/15.05.02.01/15.05.02.01-001.js]: Doesn't this work correctly without `? true : false`?

File: tests/jerry-test-suite/15/15.05/15.05.04/15.05.04.04/15.05.04.04-001.js [hunk 3]
```diff
@@ -12,4 +12,4 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-assert(String("abc").charAt(-1) === "" ? 1 : 0);
\ No newline at end of file
+assert(String("abc").charAt(-1) === "" ? true : false);
```
Comment (ruben-ayrapetyan) [inline tests/jerry-test-suite/15/15.05/15.05.04/15.05.04.04/15.05.04.04-001.js]: Likewise

File: tests/jerry-test-suite/15/15.05/15.05.04/15.05.04.04/15.05.04.04-002.js [hunk 4]
```diff
@@ -12,4 +12,4 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-assert(String("abc").charAt(3) === "" ? 1 : 0);
\ No newline at end of file
+assert(String("abc").charAt(3) === "" ? true : false);
```
Comment (ruben-ayrapetyan) [inline tests/jerry-test-suite/15/15.05/15.05.04/15.05.04.04/15.05.04.04-002.js]: Likewise. And the same below, several times.

**Comment by ruben-ayrapetyan** [comment]
@LaszloLango, generally the change is good for me. Please, remove redundant `? true : false`.

**Comment by LaszloLango** [comment]
@ruben-ayrapetyan, you are right. The simple string replace was not the best idea. :) I've updated the PR, please check.

**Comment by ruben-ayrapetyan** [comment]
Looks good to me

**Comment by zherczeg** [comment]
LGTM

**Pull request merged**